handlers/dashboard.go

```diff
@@ -443,8 +443,33 @@ func getNextWithdrawalRow(queryValidators []uint64) ([][]interface{}, error) {
 	return nextData, nil
 }
 
+func parseDayParam(param string) (uint64, error) {
+	if len(param) == 0 {
+		return 0, nil
+	}
+	val, err := strconv.ParseUint(param, 10, 23)
```

Comment:
I guess the 23 is a typo and should be 32?

Reply:
It was not a typo. But now that I have reworked the endpoint I don't need this anymore so I changed it to 32.
It was not a typo tho...